### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by Itachii27**
Fixed issue #4

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (Devanik21) [inline CODE_OF_CONDUCT.md:1]: The Galactic Mining Hub is an innovative Machine Learning-based platform 

Is it a platform or the project name?

Comment (Devanik21) [reply CODE_OF_CONDUCT.md:1]: > The Galactic Mining Hub is an innovative Machine Learning-based platform
> 
> Is it a platform or the project name?

can we assign another name to the community ?

Comment (Devanik21) [reply CODE_OF_CONDUCT.md:1]: please review and lets make it more informative

Comment (Itachii27) [reply CODE_OF_CONDUCT.md:1]: Hi @Devanik21 The Introduction of this repository states it's **Galactic Mining Hub**. If you want other community name, kindly let me know.

**Comment by Devanik21** [comment]
Hi, any innovative, futuristic name ?

**Comment by Itachii27** [comment]
> Hi, any innovative, futuristic name ?

OrbitRift or AstroDig?

**Comment by Devanik21** [comment]
> > Hi, any innovative, futuristic name ?
> 
> OrbitRift or AstroDig?

And u can modify it too for flexibility,friendly tone

**Comment by shravya312** [comment]
is StarMine name fine? @Devanik21

**Comment by Devanik21** [comment]
@shravya312  fine

**Comment by shravya312** [comment]
I want to understand the whole project clearly and contribute can u please share the research paper or any source @Devanik21

**Comment by Devanik21** [comment]
Please read the discussion, learn how each page works in app.
I have attached a link of the app

File: CODE_OF_CONDUCT.md
Comment (Devanik21) [reply CODE_OF_CONDUCT.md:1]: @Itachii27  u can raise new PR with thsese updates.

**Comment by Itachii27** [comment]
Hi @Devanik21 I have done changes. Please check and let me know.

**Pull request merged**